src/control/cmd/dmg/pretty/storage_nvme_test.go

```diff
@@ -21,6 +21,49 @@ import (
 	"github.com/daos-stack/daos/src/control/server/storage"
 )
 
+func TestPretty_PrintNVMeController(t *testing.T) {
+	for name, tc := range map[string]struct {
+		devices     storage.NvmeControllers
+		expPrintStr string
+	}{
+		"multiple controllers": {
+			devices: storage.NvmeControllers{
+				storage.MockNvmeController(1),
+				storage.MockNvmeController(2),
+			},
+			expPrintStr: `
+NVMe PCI     Model   FW Revision Socket ID Capacity 
+--------     -----   ----------- --------- -------- 
+0000:80:00.1 model-1 fwRev-1     1         2.0 TB   
+0000:80:00.2 model-2 fwRev-2     0         2.0 TB   
+`,
+		},
+		"vmd backing devices": {
+			devices: storage.NvmeControllers{
+				&storage.NvmeController{PciAddr: "050505:01:00.0"},
+				&storage.NvmeController{PciAddr: "050505:03:00.0"},
+			},
+			expPrintStr: `
+NVMe PCI       Model FW Revision Socket ID Capacity 
+--------       ----- ----------- --------- -------- 
+050505:01:00.0                   0         0 B      
+050505:03:00.0                   0         0 B      
```

Comment:
Is this what the output actually looks like when VMD is enabled? Nothing except the PCI address? If I were an admin looking at this, I would be confused.

Reply:
the test case just needed to verify the printing of the leading zero in the address, so I didn't populate the other details that's why they are not printed, normally device details would be printed for backing devices if VMD is enabled

Reply:
Okay, that works. Thanks for the explanation.